src/pages/guidelines/color/implementation.mdx

```diff
@@ -0,0 +1,546 @@
+---
+title: Color
+description:
+  Maintaining consistent and engaging digital interfaces throughout IBM, whether
+  applications or experiences, demands extended guidance around color usage.
+tabs: ['Overview', 'Usage', 'Code', 'Implementation']
+---
+
+<PageDescription>
+
+Implementing color in Carbon has never been easier. We've provided users with
+several techniques to implement all your product's color and theming needs.
+
+</PageDescription>
+
+<AnchorLinks>
+
+<AnchorLink>Layering model tokens</AnchorLink>
+<AnchorLink>Inline theming</AnchorLink>
+<AnchorLink>Light or dark mode</AnchorLink>
+
+</AnchorLinks>
+
+## Layering model tokens
+
+There are two ways to implement the
+[layering model](/guidelines/color/overview#layering-model) within a single
+theme, either by using explicit layer tokens or smart contextual layer tokens.
+Both ways are acceptable with different pros and cons that should be weighed in
+deciding which to use when building your product.
+
+| Token type                                                                          | Definition                                                                                                                                                                                             |
+| ----------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| [Layer set tokens](/guidelines/color/implementation#layer-set-tokens)               | Explicit tokens used to manually map the layering model onto components. They come in sets based on their level in the layering model and are part of the theme package.                               |
+| [Contextual layer tokens](/guidelines/color/implementation#contextual-layer-tokens) | Abstracted tokens that automatically infer value based on the context a component inside the layering model. They are part of the component package and don't have an explicit value assigned to them. |
```

Comment:
> They are part of the component package and don't have an explicit value assigned to them.

I would just drop this line when talking about which package they belong in. There are several ways people could get access to them so may not be helpful specifying a single package.